src/components/general/NotificationsSideSheet/index.tsx

```diff
@@ -13,59 +15,31 @@ type NotificationsSideSheetProps = {
 };
 
 type NotificationsSideSheetContentProps = {
+    settings: React.ReactElement | null;
+    showSettings?: boolean;
     readNotifications: NotificationCard[];
     unReadNotifications: NotificationCard[];
     isFetchingReadNotifications: boolean;
     isFetchingUnReadNotifications: boolean;
 };
 
 const NotificationsSideSheetContent: React.FC<NotificationsSideSheetContentProps> = ({
```

Comment:
this file contains two components, there should only be one/file, split into 2 files

Reply:
split the two components into two files